extensions/cornerstone/src/utils/WarningsViewportOverlay.js

```diff
@@ -0,0 +1,236 @@
+import { PureComponent } from 'react';
+import React from 'react';
+import PropTypes from 'prop-types';
+import cornerstone from 'cornerstone-core';
+import './WarningsViewportOverlay.css';
+import moment from 'moment';
+import classNames from 'classnames';
+import { Icon } from '@ohif/ui/src/elements/Icon';
+import { Tooltip } from '@ohif/ui/src/components/tooltip';
+import { OverlayTrigger } from '@ohif/ui/src/components/overlayTrigger';
+
+function isValidNumber(value) {
+  return typeof value === 'number' && !isNaN(value);
+}
+
+function formatNumberPrecision(number, precision) {
+  if (number !== null) {
+    return parseFloat(number).toFixed(precision);
+  }
+}
+
+/**
+* Formats DICOM date.
+*
+* @param {string} date
+* @param {string} strFormat
+*/
+function formatDICOMDate(date, strFormat = 'MMM D, YYYY') {
+  return moment(date, 'YYYYMMDD').format(strFormat);
+}
+
+/**
+*    DICOM Time is stored as HHmmss.SSS, where:
+*      HH 24 hour time:
+*        m mm        0..59   Minutes
+*        s ss        0..59   Seconds
+*        S SS SSS    0..999  Fractional seconds
+*
+*        Goal: '24:12:12'
+*
+* @param {*} time
+* @param {string} strFormat
+*/
+function formatDICOMTime(time, strFormat = 'HH:mm:ss') {
+  return moment(time, 'HH:mm:ss').format(strFormat);
+}
+
+/**
+ * Formats a patient name for display purposes
+ */
+function formatPN(name) {
```

Comment:
I think you can extract all these utils to a utils file in utils folder and move the component to a components folder

Reply:
done